### PR TITLE
Fix Monitor Integration Test Flake: added wait until expected text value to be present

### PR DESCRIPTION
**Pull request by dtaylor113**

### CODE REVIEW

File: frontend/integration-tests/tests/monitoring.scenario.ts [hunk 1]
```diff
@@ -245,7 +245,10 @@ describe('Alertmanager: Configuration', () => {
     await firstElementByTestID('input-repeat-interval').sendKeys('24h');
 
     await monitoringView.saveButton.click();
-    await crudView.isLoaded();
+    browser.wait(
+      until.textToBePresentInElement(firstElementByTestID('repeat_interval_value'), '24h'),
+      5000,
```
Comment (spadgett) [inline frontend/integration-tests/tests/monitoring.scenario.ts]: I'd leave off the custom timeout... Just use the default

**Comment by benjaminapetersen** [comment]
Frontend change looks good, still have this for backend:

```
go: finding google.golang.org/genproto v0.0.0-20190819201941-24fa4b261c55
/go/pkg/mod/github.com/coreos/pkg@v0.0.0-20180928190104-399ea9e2e55f/capnslog/journald_formatter.go:25:2: no matching versions for query "latest"
```

File: frontend/integration-tests/tests/monitoring.scenario.ts (hunk 1 above)
Comment (dtaylor113) [reply frontend/integration-tests/tests/monitoring.scenario.ts]: ok, removed the 5000

**Comment by dtaylor113** [comment]
Hi @spadgett, not sure what to do with this PR :worried:

**Comment by benjaminapetersen** [comment]
I think its uncertain if this is needed now.  We aren't seeing the flake anymore elsewhere.

**Comment by dtaylor113** [comment]
> I think its uncertain if this is needed now. We aren't seeing the flake anymore elsewhere.

Hi, @benjaminapetersen thought [this](https://github.com/openshift/console/pull/4041#issuecomment-578132691) was another flake, I'll retest.

**Comment by dtaylor113** [comment]
Not needed

**Pull request closed without merging**